projects/admin/src/app/service/library.service.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Injectable } from '@angular/core';
+import { RecordService } from '@rero/ng-core';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LibraryService {
+
+  static readonly resource = 'libraries';
+
+  constructor(
+    private recordService: RecordService
+  ) { }
+
+  get count$() {
+    return this.recordService.getRecords(
+      LibraryService.resource,
+      undefined,
+      undefined,
+      0
+    ).pipe(
+      map(results => results.hits.total)
+    );
+  }
+
+  currentLibrary$(pid: string) {
+    return this.recordService.getRecord('libraries', pid);
+  }
```

Comment:
Why `currentLibrary$(pid: string)` instead of `get$(pid: string)` ?
For me you can implements both but `currentLibrary$` could be better as 
```
currentLibrary$() {
    const user = userService.getCurrentUser();
    return this.recordService.getRecord('libraries', user. currentLibrary);
}
```

Reply:
done